it/__init__.py

```diff
@@ -29,7 +29,7 @@
 from esrally.utils import process
 
 CONFIG_NAMES = ["in-memory-it", "es-it"]
-DISTRIBUTIONS = ["5.6.16", "6.8.0", "7.6.0"]
+DISTRIBUTIONS = ["6.8.0", "7.6.0"]
```

Comment:
We're doing this here (as well) to decouple this PR from https://github.com/elastic/rally/pull/1237 as rally-tracks still uses this feature for 5.x clusters.